concepticondata/conceptlists/Bowern-2017-348.tsv

```diff
@@ -54,7 +54,7 @@ NUMBER	ID	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
 53	Bowern-2017-348-53	boil/pimple	1809	PIMPLE
 54	Bowern-2017-348-54	bone	1394	BONE
 55	Bowern-2017-348-55	boomerang/throwing stick	135	BOOMERANG
-56	Bowern-2017-348-56	bottom grinding stone		
+56	Bowern-2017-348-56	bottom grinding stone	2459	GRINDSTONE
```

Comment:
Good catch, @Tarotis. There are also other bottom grindstones mapped to GRINDSTONE. @mathildavz Could you please open an issue that we check the GRINDSTONE mappings? You can mention this PR so that we know that we also check the Bowern list.

Reply:
Thank you for catching that @Tarotis! I'll open an issue about it.